### DISCUSSION TITLE
Custom path for the JS-/CSS-files

### DISCUSSION THREAD

**Discussion by acmuehlhausen-smc** [quarto-dev/quarto-cli#3639] [Q&A] [answered]
Hey there! 

I use Quarto to create html files that I embed within a larger website. For that, I render `report.qmd` with `embed-resources: false` and copy the scripts from `report_files/` to a folder on my server. Then, I search and replace all occurences of `<script src="report_files/libs/` in the rendered html with the corresponding URL on my server. I was wondering whether there is a way to just render it out that way? I tried the param `lib-dir` as I'd have guessed it does what I want but it had no effect.

Any ideas?

Thanks for your work, really like quarto!

Best! 
Alex

**Comment by mcanouil** [comment, marked as answer]
No, there isn't right now.
I can't find it right now, but there was an issue/discussion about this.

Currently the _files is completely tied up to Quarto internals behaviour.

**Reply by acmuehlhausen-smc** [reply]
Thanks, good to know that!
